### ISSUE TITLE
Add HTTP Errors

### ISSUE THREAD

**Issue by danieljoppi** [JS-DevTools/ono#2]
some like this 

```javascript
const ono = require('ono');

ono.badRequet(); // => ono( {status: 400, message: 'Bad Request'})
ono.unauthorized(); // => ono({status: 401, message: 'Unauthorized'})
ono.forbidden(); // => ono({status: 403, message: 'Forbidden'})
ono.notFound(); // => ono({status: 404, message: 'Not Found'})
```

**Comment by BigstickCarpet** [comment]
I like this idea, but I think it should be a separate npm package (e.g. `ono-http`) rather than part of the main `ono` package.

**Comment by JamesMessinger** [comment]
We just released Ono v5.0, which allows you to define your own custom `ono` methods.  So it's now super easy to implement HTTP errors like the ones described above.  [More details here](https://github.com/JS-DevTools/ono#custom-error-classes)
